core/token.go

```diff
@@ -40,3 +45,19 @@ func getSecret(filename string) (string, error) {
 	err = json.Unmarshal(raw, &s)
 	return s.Secret.Token, err
 }
+
+func createTokenFile(u string, t string, filename string) error {
+
+	data := userTokenPair{
+		User:  u,
+		Token: t,
+	}
+
+	jdata, err := json.MarshalIndent(data, "", " ")
+	if err != nil {
+		return err
+	}
+
+	err = ioutil.WriteFile(filename, jdata, 0644)
```

Comment:
should this token be created by default with `0600` mode? As it is currently it is world-readable, which for an access token seems undesirable